pkg/controller/kubelet-config/kubelet_config_controller_test.go

```diff
@@ -184,13 +185,17 @@ func newKubeletConfig(name string, kubeconf *kubeletconfigv1beta1.KubeletConfigu
 	}
 }
 
-func (f *fixture) newController() *Controller {
+func (f *fixture) newController(fgAccess featuregates.FeatureGateAccess) *Controller {
 	f.client = fake.NewSimpleClientset(f.objects...)
 	f.oseclient = oseconfigfake.NewSimpleClientset(f.oseobjects...)
 
 	i := informers.NewSharedInformerFactory(f.client, 0)
 	featinformer := oseinformersv1.NewSharedInformerFactory(f.oseclient, 0)
 
+	if fgAccess == nil {
+		fgAccess = featuregates.NewHardcodedFeatureGateAccess(nil, nil)
+	}
```

Comment:
Some controllers don't use the fgAccess so it's ok for those ones to pass a nil through